sass/base/components/content-block/_content-block.scss

```diff
@@ -66,10 +66,7 @@
   bottom: 0;
   width: 100%;
   padding: 40px 10px 20px;
-  background-image: -webkit-gradient(linear,50% 0%,50% 100%,color-stop(0%,rgba(0,0,0,0)),color-stop(100%,#061d38));
-  background-image: -moz-linear-gradient(rgba(0,0,0,0),#061d38);
-  background-image: -webkit-linear-gradient(rgba(0,0,0,0),#061d38);
-  background-image: linear-gradient(rgba(0,0,0,0),#061d38);
+  background-image: linear-gradient(rgba(0,0,0,0), $colour-dark-blue);
```

Comment:
Are we happy that PostCSS fills in _all_ of the gaps for all browser fixes for this?

Reply:
I've checked Chrome, Safari, Firefox, IE11, Edge.. no problems with the gradient, vendor prefixes are added. 
I'm not sure what the particular use case of `-webkit-gradient(linear,50% 0%,50% 100%,color-stop(0%,rgba(0,0,0,0)),color-stop(100%,#061d38));` is though, but didn't see any issues in webkit browsers.

Reply:
https://developer.mozilla.org/en-US/docs/Web/CSS/linear-gradient - "[3] WebKit since 528 supports the **legacy** -webkit-gradient(linear,…) function."

Cool, looks like it'll be fine *thumbs-up emoji*